Documentation of the additional GQL error info in the query log (#1824)
neo4j/docs-operations: modules/ROOT/pages/monitoring/logging.adoc

| Reason for failure.
Included when applicable.

> | errorInfo

Author: Should we somehow specify that this field in only available starting from version 5.25?

| transactionId
| ID of the transaction.
|===

==== GQL error information

The GQL error information is included in the querylog as the field `errorInfo`.
The JSON object `errorInfo` can contain the following elements:

> * `GQLSTATUS:` A 5 character long alpha-numeric code which identifies the error.

Reviewer: ```suggestion
* `GQLSTATUS` -- A 5-character long alpha-numeric code identifying the error.
```

> * `statusDescription:` A message describing the error.

Reviewer: ```suggestion
* `statusDescription` -- A message describing the error.
```

> * `classification:` The type of error, representing a division of client errors, transient errors and database errors.

Reviewer: ```suggestion
* `classification` -- The type of error representing a division of client, transient, and database errors.
```

> * `position:` The position (a JSON object containing a field for `column`, `offset` and `line`) in the query where this error occurred.

Reviewer: ```suggestion
* `position` -- The position (a JSON object containing a field for `column`, `offset` and `line`) in the query where this error occurred.
```

> * `cause:` This is a JSON object containing the errorInfo JSON object of the cause of the current errorInfo JSON object.

Reviewer: ```suggestion
* `cause` -- A JSON object containing the `errorInfo` JSON object of the cause of the current `errorInfo` JSON object.
```
